recipes/tip/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "tip" %}
+{% set version = "v1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/IM-USAF/tip/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 4ba6f6a9e652fa48a99534997aaaa0921a94af0ef2d65ce909b6825c06a2b259
+
+build:
+  skip: true  # [win and vc<14 or osx]
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('tip', max_pin='x.x') }}
+  script_env: 
+    - TIP_VERSION={{ version }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - make ==4.3  # [unix]
+    - ninja ==1.10.2
+    - cmake ==3.21.3
+    - pkgconfig ==1.5.5 
+    - libpcap ==1.10.1
+    - arrow-cpp ==6.0.2
+    - spdlog ==1.8.5
+    - yaml-cpp ==0.6.3
+    - gmock ==1.10.0
+    - gtest ==1.10.0
+    - libtins ==4.2
+
+  host:
+    - libpcap ==1.10.1
+    - arrow-cpp ==6.0.2
+    - spdlog ==1.8.5
+    - yaml-cpp ==0.6.3
+    - gmock ==1.10.0
+    - gtest ==1.10.0
+    - libtins ==4.2
+
+  run:
+    - libpcap ==1.10.1
+    - arrow-cpp ==6.0.2
+    - spdlog ==1.8.5
+    - yaml-cpp ==0.6.3
+    - gmock ==1.10.0
+    - gtest ==1.10.0
+    - libtins ==4.2
```

Comment:
Why do you want/need to pin to the minor version? Pinning to minor versions is usually not necessary because these packages probably define their own run exports which define their ABI compatability. If you're unsure, you can manually check their feedstocks.

Reply:
We've been creating a TIP conda package in our private CI for about a year and have run into issues at env solve time, particularly related to arrow-cpp, when packages are not pinned. I don't recall if we've tried pinning only the major version. I'm quite new to conda (and didn't configure the conda-build in our CI) so I'm willing to give it a try based on your expert advice.

Reply:
The update I pushed pins only the major versions.